### DISCUSSION TITLE
哥，为什么我这里一直说未知代码啥的啊

### DISCUSSION THREAD

**Discussion by csr050801** [mikumifa/cppTickerBuy#20] [提问专区]
我看上面也有人问了这个，大佬你没回答
![Uploading 微信截图_20240926202501.png…]()

**Comment by mikumifa** [comment]
图没传出来，GitHub要等一会

**Reply by csr050801** [reply]
![微信截图_20240926202501](https://github.com/user-attachments/assets/2ddbfbba-c8b1-4aa6-a3a3-237e4f644609)
我发现我图片有点传不上去，要不哥你看看我前面第三个兄弟的问题，我和他一样的

**Reply by csr050801** [reply]
啊传上去了

**Comment by mikumifa** [comment]
你抢的事哪个票 有网站吗，我复现一下

**Reply by csr050801** [reply]
https://www.allcpp.cn/allcpp/event/event.do?event=3724
其实我已经试过两个了，这是其中一个，两个都是这个问题

**Reply by csr050801** [reply]
https://www.allcpp.cn/allcpp/event/event.do?event=3677这是另外一个

**Reply by csr050801** [reply]
浏览器用过edge和谷歌的那个了都是这个样子，我有在考虑会不会是我电脑问题了

**Comment by mikumifa** [comment]
完蛋，这个好像是cpp的风控，你换个梯子节点应该就能用了，跟脚本没有关系，因为你去浏览器估计也买不了票
ip被封了

**Reply by csr050801** [reply]
啊还有这种事，但是大佬你不是说开梯子会出问题嘛，扎心了

**Reply by mikumifa** [reply]
忘记了，梯子也是有许多类型的

**Reply by mikumifa** [reply]
有些梯子就没事

**Reply by mikumifa** [reply]
风控救不了，只能降低速度

**Reply by csr050801** [reply]
好的好的这种情况如果直接换台电脑行不行啊

**Reply by csr050801** [reply]
我试着电脑换成热点没有使用校园网成功拿下，感谢大佬写出来的东西
